nebula/ui/themes/themes.js

```diff
@@ -200,6 +200,12 @@ theme.vpnToggleDisconnected = {
   'focusBorder': theme.darkFocusBorder,
 };
 
+theme.card = {
+    'defaultColor': theme.white,
+    'focusOutline': theme.bgColorTransparent,
+    'focusBorder': theme.bgColorTransparent,
```

Comment:
Do we know what should happen when a card has active focus or is hovered/clicked? I don't think I've seen it called out in the spec.